parquet/src/main/java/org/apache/iceberg/parquet/TypeWithSchemaVisitor.java

```diff
@@ -149,6 +134,50 @@
     }
   }
 
+  private static <T> T visitTwoLevelList(
+      org.apache.iceberg.types.Type iType,
+      TypeWithSchemaVisitor<T> visitor,
+      GroupType group,
+      Type repeatedElement) {
+    Types.ListType list = null;
+    Types.NestedField element = null;
+    if (iType != null) {
+      list = iType.asListType();
```

Comment:
This can be pushed up to the method call. `iType.asListType()` passed in here makes the signature of this method simpler, `Types.ListType`. Same with the other method.